### DISCUSSION TITLE
Docker pimcore, why no nginx?

### DISCUSSION THREAD

**Discussion by webdeploy-davoli-alessio** [pimcore/pimcore#7816] [Q&A / Support]
As the official documentation says:
- "Installation on Nginx is entirely possible, and in our experience quite a lot faster than apache."

So now i ask, why the official pimcore/pimcore uses apache instead of nginx?

I am now trying to create a docker compose for my use case, and nginx would be the correct choice, are there some technical impediments that you have already found? I would be glad to help on configuring a docker image with nginx!

Thanks to everyone for the good work!

**Comment by rubanooo** [comment]
I have created my own docker-compose file with nginx and the nginx config found in the pimcore docs and it works like a charm

**Reply by erkobridee** [reply]
can you share how you did that? thanks 😃

**Reply by webdeploy-davoli-alessio** [reply]
Hi, in the end i made it too combining this references:

- https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Nginx_Configuration.html
- https://github.com/markhilton/docker-php-fpm
- https://gist.github.com/md5/d9206eacb5a0ff5d6be0

**Comment by dpfaffenbauer** [comment]
because I maintain the images and I prefer apache... that is honestly the only reason.

**Reply by frithjof** [reply]
dominiks docker setup is more like one example way of running pimcore ( super nice to have it )

Documentation has all you need for an nginx setup with or without docker

**Comment by ChrisB9** [comment]
if you are searching for an alternative that is nginx based (not pimcore x compatible) - then maybe this one is for you:
https://github.com/ChrisB9/pimcore-docker
